yt_dlp/extractor/mave.py

```diff
@@ -68,40 +117,75 @@ class MaveIE(InfoExtractor):
+    }, {
+        'url': 'https://ochenlichnoe.mave.digital/',
+        'info_dict': {
+            'id': 'ochenlichnoe',
+            'title': 'Очень личное',
+            'description': 'md5:ee36a6a52546b91b487fe08c552fdbb2',
+        },
+        'playlist_mincount': 20,
+    }, {
+        'url': 'https://geekcity.mave.digital/',
+        'info_dict': {
+            'id': 'geekcity',
+            'title': 'Мужчины в трико',
+            'description': 'md5:4164d425d60a0d97abdce9d1f6f8e049',
+        },
+        'playlist_mincount': 80,
+    }]
+
+    def _real_extract(self, url):
+        channel_id = self._match_id(url)
+
+        channel_meta = self._load_channel_meta(channel_id)
+
+        entries = []
+        page = 1
+
+        while True:
+            data = self._download_json(
+                f'{self._API_BASE_URL}/{channel_id}/episodes?'
+                f'view=all&page={page}&sort=newest&format=all',
+                channel_id)
+
+            episodes = data.get('episodes', [])
+
+            if not episodes:
+                break
+            else:
+                for episode_meta in data['episodes']:
+                    entries.append(self._create_entry(
+                        channel_id, channel_meta, episode_meta))
+
+                page += 1
 
         return {
-            'display_id': display_id,
-            'channel_id': channel_id,
-            'channel_url': f'https://{channel_id}.mave.digital/',
-            'vcodec': 'none',
-            'thumbnail': re.sub(r'_\d+(?=\.(?:jpg|png))', '', self._og_search_thumbnail(webpage, default='')) or None,
-            **traverse_obj(data, ('activeEpisodeData', {
-                'url': ('audio', {urljoin(self._API_BASE_URL)}),
-                'id': ('id', {str}),
-                'title': ('title', {str}),
-                'description': ('description', {clean_html}),
-                'duration': ('duration', {int_or_none}),
-                'season_number': ('season', {int_or_none}),
-                'episode_number': ('number', {int_or_none}),
-                'view_count': ('listenings', {int_or_none}),
-                'like_count': ('reactions', lambda _, v: v['type'] == 'like', 'count', {int_or_none}, any),
-                'dislike_count': ('reactions', lambda _, v: v['type'] == 'dislike', 'count', {int_or_none}, any),
-                'age_limit': ('is_explicit', {bool}, {lambda x: 18 if x else None}),
-                'timestamp': ('publish_date', {parse_iso8601}),
-            })),
-            **traverse_obj(data, ('podcast', 'podcast', {
-                'series_id': ('id', {str}),
-                'series': ('title', {str}),
-                'channel': ('title', {str}),
-                'uploader': ('author', {str}),
-            })),
+            '_type': 'playlist',
+            'id': channel_id,
+            'title': channel_meta['podcast']['title'],
+            'description': channel_meta['podcast']['description'],
+            'entries': entries[::-1],
```

Comment:
And also make the metadata extraction non-fatal
```suggestion
            **traverse_obj(channel_meta, {
                'title': ('podcast', 'title', {str}),
                'description': ('podcast', 'description', {str}),
            }),
            'entries': InAdvancePagedList(
                functools.partial(self._entries, channel_id, channel_meta),
                math.ceil(channel_meta['podcast']['episodes_count'] / self._PAGE_SIZE), self._PAGE_SIZE),
```